src/message/index.ts

```diff
@@ -34,7 +34,9 @@ export class Message {
       content.set(2, obj.to.toString());
     }
     content.set(3, obj.method);
-    content.set(4, cbor.encode(obj.data ? obj.data : new ArrayBuffer(0)));
+    if (obj.data) {
+      content.set(4, cbor.encode(obj.data));
+    }
```

Comment:
Don't we still need an empty buffer here if there's no `data` field in `obj`? That field is required by the spec, right @hansl ?